docs/observability/how_to_guides/dashboards.mdx

```diff
@@ -3,71 +3,105 @@ sidebar_label: Dashboards
+
+| Section         | What it shows                                                                                                                                                                                                                                          |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Traces          | A [trace](/observability/concepts#traces) is a collection of [runs](/observability/concepts#runs) related to a single operation. For example, if a user request triggers an agent, all runs for that agent invocation would be part of the same trace. |
+| LLM Calls       | All runs where run type is "llm".                                                                                                                                                                                                                      |
+| Cost & Tokens   | Costs are measured using [LangSmith's cost tracking](/observability/how_to_guides/log_llm_trace#manually-provide-token-counts).                                                                                                                        |
+| Tools           | Includes runs where run type is "tool". Limits to top [N] tools.                                                                                                                                                                                       |
+| Run Types       | Includes runs that are immediate children of the root run. For example, for the following trace, the following runs have a depth of 1: <img src={runDepth} alt="run_depth" style= {{ display: 'block', width: 200 }} />. Limits to top [N] runs.       |
+| Feedback Scores | Feedback provided as part of [online evaluators](/observability/how_to_guides/online_evaluations). Limits to top the top 4 feedback scores by volume.                                                                                                  |
+
+### Group by
+
+Group by [run tag or metadata](/observability/how_to_guides/add_metadata_tags) can be used to split data over arbitrary metrics that are important to your application. The group by setting appears on the top right hand side of the dashboard. Note that the Tool and Run Type charts already have a group by applied, so the global group by won't take effect.
+
+:::tip
+When adding metadata to runs, we reccomend having the same metadata on the trace, as well has the specific run (eg. LLM call). [WHY]
+:::
+
+### Time selector
+
+Choose a time window in the dropdown on the top right hand side of the dashboard. You can set a default time window, by hovering over the values in the dropdown and clicking "Set default". Maximum time horizon is 30d.
+
+:::note
```

Comment:
feels like this is a part of the Time Selector section, which i assume its not

Reply:
maybe should have a ### Editing prebuilts (in progress) section that just has this note in it